src/helpers/send-newsletter.ts

```diff
@@ -232,21 +274,67 @@ function generateHTMLTemplate(data: NewsLetterDataType) {
         </div>
   
         <!-- New Threads This Week -->
-        ${data.new_threads_this_week.length > 0
-          ? `<h2>New Threads This Week</h2>` +
-            data.new_threads_this_week
-              .map((thread) => `<div class="card">${generateHTMLForPost(thread)}</div>`)
-              .join("")
-          : '<p>No new threads this week.</p>'}
+        ${
+          data.new_threads_this_week.length > 0
+            ? `<h2>New Threads This Week</h2>` +
+              data.new_threads_this_week
+                .map(
+                  (thread) =>
+                    `<div class="card">${generateHTMLForPost(thread)}</div>`
+                )
+                .join("")
+            : "<p>No new threads this week.</p>"
+        }
+
+        <!-- Survey Button -->
+        <div style="text-align: center; padding: 24px;">
+         <div class="iterateQuestionPrompt" style="font-size: 16px; line-height: 24px; text-align: center; margin-bottom: 10px;">
+            <p style="text-align: center;">Did you find the information in this email helpful?</p>
+         </div>
+         <table style="margin: 0 auto; text-align: center;">
+            <tr>
+                <td><a href="https://us17.list-manage.com/survey?u=718f9c0ab4af9b4acf93a8e6f&id=c2c0b80d59&attribution=false" style="text-decoration: none; color: inherit;"><span style="font-size: 36px; padding: 8px;">😒</span></a></td>
```

Comment:
This code is repeated 8 times, why not add the emojis in a table and iterate over it? Repeating code is never a good idea.

Reply:
Yeah, I get it, this was PoC for changing from button to emoji

Reply:
If it's on an open Pull Request and ready to be merged with the main codebase, then it's not a PoC imo.